### PR TITLE
gofmt

### PR DESCRIPTION
**Pull request by robx**
Keeps showing up when I gofmt our whole tree including godeps.

### CODE REVIEW

**Comment by fabxc** [comment]
Thanks. Could you move it back down there but insert a blank line above?

**Comment by robx** [comment]
Sure. There you go.

File: text/bench_test.go [hunk 1]
```diff
@@ -19,6 +19,7 @@ import (
 	"io"
 	"io/ioutil"
 	"testing"
+
 	dto "github.com/prometheus/client_model/go"
```
Comment (beorn7) [inline text/bench_test.go]: To fix this properly (according to the Prometheus style guide) this has to go to the bottom most group.

(The original placing was a glitch, and older gofmt liked it there.)

Comment (robx) [inline text/bench_test.go]: Hah, I wanted to do this, but then I saw it was different in other places.

**Comment by fabxc** [comment]
Thanks!

**Pull request merged**